.github/workflows/ci.yml

```diff
@@ -363,6 +363,7 @@ jobs:
         version:
           [
             "23.8.11.29.altinitystable",
```

Comment:
Should this `version` be left here or should it be removed? I asked this because no previous versions are present in this `version` list.